components/connect/components/search/view.coffee

```diff
@@ -15,8 +15,14 @@ module.exports = class ConnectSearchView extends Backbone.View
     input: ->
       @query arguments...
 
-  initialize: ({ @state, @search }) ->
+  initialize: ({ @state, @search, @eventBus }) ->
     @query = throttle @perform, 200
+    
+    @eventBus.on 'connection:added', @afterConnection
+
+  afterConnection: =>
+    @$el.focus()
+    @$el.select()
```

Comment:
Yeah, looks like you used it out of the context of a view, makes sense https://github.com/aredotna/ervell/blob/master/components/connect/client/index.coffee